client/layout/masterbar/logged-in.jsx

```diff
@@ -17,6 +17,9 @@ import config from 'config';
 import { preload } from 'sections-preload';
 import ResumeEditing from 'my-sites/resume-editing';
 import { setNextLayoutFocus } from 'state/ui/layout-focus/actions';
+import { decorate } from 'plugins/helpers';
+
+const preloadStats = preload.bind( null, 'stats' );
```

Comment:
I don't think it belongs in this PR, but we want to avoid binds in render calls for perf reasons.